src/main/java/com/internet/shop/db/Storage.java

```diff
@@ -0,0 +1,39 @@
+package com.internet.shop.db;
+
+import com.internet.shop.model.Order;
+import com.internet.shop.model.Product;
+import com.internet.shop.model.ShoppingCart;
+import com.internet.shop.model.User;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<Product> products = new ArrayList<>();
+    public static final List<Order> orders = new ArrayList<>();
+    public static final List<ShoppingCart> shoppingCarts = new ArrayList<>();
+    public static final List<User> users = new ArrayList<>();
+    private static Long productId = 0L;
+    private static Long orderId = 0L;
+    private static Long shoppingCartId = 0L;
+    private static Long userId = 0L;
+
+    public static void addProduct(Product product) {
+        product.setId(productId++);
```

Comment:
```suggestion
        product.setId(++productId);
```
ids should start with 1